README.md

```diff
@@ -1,62 +1,36 @@
-# 0g-DA-signer
-
-## Download Encoding Params
```

Comment:
It's better to also maintain the original guidance regarding installation.

Reply:
agree

Reply:
agree

Reply:
hey @csdtowards @MiniFrenchBread , are there instructions that are inherently wrong about this new README?

because for now, we are trying to move towards the convention that the repo README's do not offer detailed instruction, and instead direct developers to new documentation link. because we have so many products this ensures a single source of truth. 

happy to keep instructions on README. but we need one way across all repos, so that our team know when to update what. 

if we should find a time to discuss, happy to do it. thank you so much!